.github/workflows/nightly.yml

```diff
@@ -0,0 +1,34 @@
+name: Upload Nightly Python Package Release
+
+on: 
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Normalized Version Tag'     
+        required: true
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Run version checks
+      run: |
+        python release.py nightly ${{ github.event.inputs.version }}  # check matching nightly versions
+    - name: Build and publish to PyPI package 'epispot-nightly'
+      env:
+        TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
+      run: |
+        python setup-nightly.py sdist bdist_wheel
+        twine upload dist/*
```

Comment:
Have you tested the workflow to make sure it works? Perhaps with a test PyPI package? If you haven't, I highly suggest you do.

Reply:
I've tested each of the newly-added bash scripts that will run on the GitHub shell, but we can't test the `inputs` and environment variables locally. Can you verify that those will work?